packages/legacy-workbench/src/js/angular/guides/steps/complex/enable-autocomplete/plugin.js

```diff
@@ -46,17 +43,49 @@ PluginRegistry.add('guide.step', [
                             // Remove ths listener from element. It is important when step is hided.
                             $(autocompleteCheckboxSelector).off('mouseup.autocompleteCheckboxClick');
                         }
-                    }, options)
-                },
+                    }
+                }
+
+            ]
+        }
+    },
+    {
+        guideBlockName: 'wait-for-indexing-end',
```

Comment:
The name of the step is misleading. When I see it, I expect that the step will wait until indexing ends, but actually it just shows the indexing status element.